packages/pds/src/index.ts

```diff
@@ -99,7 +99,13 @@ export class PDS {
     app.use(loggerMiddleware)
     app.use(compression())
     app.use(authRoutes.createRouter(ctx)) // Before CORS
-    app.use(cors({ maxAge: DAY / SECOND }))
+    app.use(
+      cors({
+        maxAge: DAY / SECOND,
+        credentials: true,
+        origin: (origin, callback) => callback(null, origin),
```

Comment:
want to make sure these are intended changes?

Reply:
Yes and no. Will move in separate PR.